kmip/pie/api.py

```diff
@@ -25,6 +25,51 @@ class KmipClient:
     The KmipClient provides a simple external interface for various KMIP
     operations and composes the bulk of the PyKMIP Pie API.
     """
+
+    UNIQUE_IDENTIFIER = "uid"
+    NAME = "name"
+    OBJECT_TYPE = "object_type"
+    CRYPTOGRAPHIC_ALGORITHM = "cryptographic_algorithm"
+    CRYPTOGRAPHIC_LENGTH = "cryptographic_length"
+    CRYPTOGRAPHIC_PARAMETERS = "cryptographic_parameters"
+    CRYPTOGRAPHIC_DOMAIN_PARAMETERS = "cryptographic_domain_parameters"
+    CERTIFICATE_TYPE = "certificate_type"
+    CERTIFICATE_LENGTH = "certificate_length"
+    X509_CERTIFICATE_IDENTIFIER = "x509_certificate_identifier"
+    X509_CERTIFICATE_SUBJECT = "x509_certificate_subject"
+    X509_CERTIFICATE_ISSUER = "x509_certificate_issuer"
+    CERTIFICATE_IDENTIFIER = "certificate_identifier"
+    CERTIFICATE_SUBJECT = "certificate_subject"
+    CERTIFICATE_ISSUER = "certificate_issuer"
+    DIGITAL_SIGNATURE_ALGORITHM = "digital_signature_algorithm"
+    DIGEST = "digest"
+    OPERATION_POLICY_NAME = "operation_policy_name"
+    CRYPTOGRAPHIC_USAGE_MASK = "cryptographic_usage_mask"
+    LEASE_TIME = "lease_time"
+    USAGE_LIMITS = "usage_limits"
+    STATE = "state"
+    INITIAL_DATE = "initial_date"
+    ACTIVATION_DATE = "activation_date"
+    PROCESS_START_DATE = "process_start_date"
+    PROTECT_STOP_DATE = "protect_stop_date"
+    DEACTIVATION_DATE = "deactivation_date"
+    DESTROY_DATE = "destroy_date"
+    COMPROMISE_OCCURRENCE_DATE = "compromise_occurrence_date"
+    COMPROMISE_DATE = "compromise_date"
+    REVOCATION_REASON = "revocation_reason"
+    ARCHIVE_DATE = "archive_date"
+    OBJECT_GROUP = "object_group"
+    FRESH = "fresh"
+    LINK = "link"
+    APPLICATION_SPECIFIC_INFORMATION = "application_specific_information"
+    CONTACT_INFORMATION = "contact_information"
+    LAST_CHANGE_DATE = "last_change_date"
+    CUSTOM_ATTRIBUTE = "custom_attribute"
+    ALTERNATIVE_NAME = "alternative_name"
+    KEY_VALUE_PRESENT = "key_value_present"
+    KEY_VALUE_LOCATION = "key_value_location"
+    ORIGINAL_CREATION_DATE = "original_creation_date"
```

Comment:
I added these here. We can reuse them when we define the locate function on the api.